main.go

```diff
@@ -517,9 +509,35 @@ func _main(ctx context.Context) error {
 		}
 	}
 
+	// NewAgendasDB creates an on-chain db instance the helps to store and
+	// retrieves on-chain agendas data.
+	onChainInstance, err := onchaindb.NewAgendasDB(filepath.Join(cfg.DataDir, cfg.OnChainDBFileName))
+	if err != nil {
+		return fmt.Errorf("failed to create new on-chain db instance: %v", err)
+	}
+
+	// CheckOnChainUpdates checks for onchain db updates.
+	if err = onChainInstance.CheckOnChainUpdates(dcrdClient); err != nil {
+		return fmt.Errorf("updating off chain db failed: %v", err)
+	}
+
+	// NewProposalsDB creates an off-chain db instance that helps to store and
+	// retrieve off-chain proposals data.
+	offChainInstance, err := offchaindb.NewProposalsDB(cfg.PoliteiaAPIURL,
+		filepath.Join(cfg.DataDir, cfg.OffChainDBFileName))
+	if err != nil {
+		return fmt.Errorf("failed to create new off-chain db instance: %v", err)
+	}
+
+	// CheckOffChainUpdates checks for offchain db updates.
```

Comment:
The comments in the above block of code are written like function docs.  Here at the call sites in main they need to explain what they are doing and why.

e.g.

```go
// Load or create a new agenda DB.
```

```go
// Refresh agenda data from the latest information from dcrd.  This is also done when... .
```

```go
// Load or create a new Politeia proposal DB.
```

etc.